src/pf.c

```diff
@@ -76,6 +79,56 @@ static void pf_submit(char const *type, char const *type_instance, uint64_t val,
   plugin_dispatch_values(&vl);
 } /* void pf_submit */
 
+#ifdef __FreeBSD__
+static int pf_read(void) {
+  struct pfctl_status *state;
+  int fd;
+
+  fd = open(pf_device, O_RDONLY);
```

Comment:
Optional coding style improvement.

```suggestion

  int fd = open(pf_device, O_RDONLY);
```